### ISSUE TITLE
show collectives files?

### ISSUE THREAD

**Issue by jakobroehrl** [nextcloud/notes-android#1658]
Please use **[GitHub reactions](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/)** 👍 to show that you are affected by the same issue. Please don't comment if you have no relevant information to add!

I know, the collectives wiki files are available in the nextcloud android files app. 
But I would use the collectives wiki even more and use it for my personal notes too, if there would be a nice android app for this.
And it's almost possible to use this now if the user switches the folder path in the app from notes to collectives
![image](https://user-images.githubusercontent.com/24530680/215356061-3b7d98a2-789e-4e1a-988d-035cccfe696c.png)

**Comment by tobiasKaminsky** [comment]
@juliushaertl do you know how if this could lead to any sync/collaborative editing problems?

**Comment by juliushaertl** [comment]
It could lead to regular file conflicts when text/collectives editor is used and in the same time someone sync a file through the android client. Other than that this would work fine (I'm actually using that on my personal instance).

**Comment by tobiasKaminsky** [comment]
Thanks, Julius!
So we can think how to change between Notes and Collectives, but first would need to check with Andy, Frank, Jan if this is something we want at all.
(as currently I understood that Notes is about your own notes. So this might then lead to confusion…)

@AndyScherzinger @jancborchardt

**Comment by AndyScherzinger** [comment]
If it can be done reliably and not breaking things as mentioned by Julius I would say it is desirable. So looking forward also to Jan's take.

**Comment by tobiasKaminsky** [comment]
If we do it, then we should have a nice integration:
- capabilities give folder for Notes and Collectives (if installed)
- in Drawer a toggle between Notes and Collectives
  - changes header name, icon
  - adjusts sidebar (e.g. remove deleted notes)

**Comment by AlvaroBrey** [comment]
Won't it be confusing to have two separate server apps handled by the same android app? That is named after only one of them?

**Comment by tobiasKaminsky** [comment]
Might be…
Stupid idea: have two apps, notes and collectives, which are 99,5% the same :see_no_evil: 
(not meant serious)

**Comment by juliushaertl** [comment]
In the upfront discussions with @jancborchardt we kind of came to the conclusion to keep the separation between collectives and notes as collectives targets more the collaborative and shared aspect while notes is really focussing on personal note taking. But let's see if there is more input from him :)

**Comment by jancborchardt** [comment]
Picking this up again – so I agree of course with @jakobroehrl’s main point that it would be nice to have a mobile app for Collectives.

Now this Notes-app-hack is one way of doing it, but it seems off of course. I see 2 main directions:
- The idea @tobiasKaminsky mentioned, having two mobile apps which are nearly the same. One called Notes, one Collectives
- Or just having both in one, namely the current Notes app. There could be sections for your own Notes, and then for each of the collectives.

And this (at least option 2) also brings up the discussion of possibly merging the web apps of Collectives and Notes. From a user perspective, having so many different apps to install is for sure confusing. Whether it’s a collection of Notes you happen to be able to share (or not) is not that big of a difference.
But of course there are more fundamental things different, like Circles usage etc. However again – not relevant in the resulting UX, so maybe we find a way to combine it. @juliushaertl

**Comment by stefan-niedermann** [comment]
> From a user perspective, having so many different apps to install is for sure confusing.

Actually currently there is one client companion per server app - Files, Talk, Notes (Deck, News, ...), so it might be more confusing if one Android app suddenly handles more than one server pendant IMHO.

It also makes the Notes app more error prone (technically one  needs to check whether both apps are installed, enabled, handle different API versions, ...) and more bloated. It contacarates the (assumed) goal to provide a blank small little note taking app. I assume this because if this wasn't a goal you could also implement Notes and Collectives into the Files app.

Just my 2ct 😉

**Comment by jancborchardt** [comment]
Yeah, good point @stefan-niedermann! I also tend more towards separate apps, as it is also a simple mental model then.

**Comment by tobiasKaminsky** [comment]
Having two apps that maybe in background share 98% of code is fine, I think.

From UX it is also better, as then both apps can be used side by side.

**Comment by siccovansas** [comment]
I really like Collectives, both in professional and personal settings, and I'm using it more and more. It works great on desktop, but for mobile an app + widget would make it much faster and easier to use. Especially when it has to compete against apps like Notion (gap analysis @jospoortvliet).

It has been 2 years since the last comment on this issue. Will a Collectives app be made using this Notes app? Or would [Neon](https://github.com/nextcloud/neon/issues/587) be used?

(for discoverability: the [link to the related issue](https://github.com/nextcloud/collectives/issues/748) in the Collectives repo).
